platform/admin/configurator/admin-configurator-impl/src/main/java/org/codice/ddf/admin/configurator/impl/BundleOperation.java

```diff
@@ -129,4 +152,30 @@ private BundleStateService getBundleStateService() {
     private boolean lookupBundleState() {
         return getBundleStateService().getState(bundle) == BundleState.Active;
     }
+
+    private boolean isPermittedToViewBundle(Bundle bundle) {
+        ServiceReference<?>[] services = bundle.getRegisteredServices();
+        if (services == null || services.length == 0) {
+            return true;
+        }
+
+        for (ServiceReference<?> service : services) {
+            Object pidProperty = service.getProperty("service.pid");
+            if (pidProperty instanceof String) {
```

Comment:
Just curious if we need this instanceof check. Will the service.pid ever be anything other than a string?